projects/aca-content/src/lib/store/effects/node.effects.spec.ts

```diff
@@ -564,4 +566,40 @@ describe('NodeEffects', () => {
       expect(store.dispatch).toHaveBeenCalledWith(new NavigateUrlAction('personal-files/details/node-id?location=test-page'));
     });
   });
+
+  describe('folderInformation$', () => {
+    it('should call folder information dialog', () => {
+      const node: any = { entry: { isFile: true } };
+      spyOn(contentService, 'showFolderInformation').and.stub();
+
+      store.dispatch(new FolderInformationAction(node));
+
+      expect(contentService.showFolderInformation).toHaveBeenCalled();
```

Comment:
can you use toHaveBeenCalledWith to test also parameters passed to showFolderInformation function?

Reply:
Updated unit test